.github/workflows/check-test.yaml

```diff
@@ -95,12 +87,26 @@ jobs:
         shell: bash
 
       - name: Run Checks
-        # We exclude dependents when running eslint and depchecks because
-        # linting and dependency checking is relevant only for packages where
-        # the source code was changed
         run: npm run check-ci -- --stream
         shell: bash
 
       - name: Run Tests
         run: npm run test-ci -- --stream
         shell: bash
+
+      - name: Report Coverage
```

Comment:
fyi: coveralls does have an official github action that could be worth considering, didn't look too much into it though, might not fit our setup: https://github.com/marketplace/actions/coveralls-github-action

Reply:
Yeah, I looked at it, but the problem with it is that it's designed to report the results of a single package run, so I'd have to either add 20 steps to report coverage from each of the packages in the monorepo, or completely change how we're running CI to make it run one job per package in a matrix. The official action is using the CLI like we are and I did consult with its implementation when working on our reporting.